src/ItemForm/index.js

```diff
@@ -1,22 +1,14 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import Form from 'react-bootstrap/Form';
 import Button from 'react-bootstrap/Button';
 import Modal from 'react-bootstrap/Modal';
 
 const ItemForm = (props) => {
-
-  const [ description, setDescription ] = useState('');
+  const type = props.type.toLowerCase();
+  const [ description, setDescription ] = useState(props.description);
   const [ isValidated, setIsValidated ] = useState(false);
-  const [ title, setTitle ] = useState('');
-
-  useEffect (() => {
```

Comment:
This was causing a console.error about mixing controlled and uncontrolled components. I've tested this code with Firebase, and it seems to be fine with setting the title and description from props, rather than the useEffect hook.